src/makeRobotAccountant.js

```diff
@@ -6,7 +6,21 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+
+  return function scope(arg1) {
+    return function getSum(arg2) {
+      const sum = arg1 + arg2;
+
+      counter++;
+
+      if (counter > 3 && counter % 2 === 0) {
+        return 'Bzzz... Error!';
+      }
+
+      return sum;
```

Comment:
```suggestion
      return arg1 + arg2;
```
No need in variable `sum`